scaffolder-templates/backstage-system-template/skeleton/all.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: backstage.io/v1alpha1
+kind: Location
+metadata:
+  name: entities
```

Comment:
We already have a location called `entities` in Janus Showcase, so this will conflict with that, and it will be unusable with the default setup of the showcase. 

Not sure if it is a problem or not.

Reply:
Good point, I'll update this.